modules/home-manager/lazygit.nix

```diff
@@ -22,10 +40,16 @@ in
     accentSupport = true;
   };
 
-  config = lib.mkIf cfg.enable {
-    # TODO: Find a way to "source" a config file with $LG_CONFIG_FILE
-    # and *not* come across https://github.com/catppuccin/nix/issues/455
-    # https://github.com/jesseduffield/lazygit/blob/4b4d82e13c568f3b43c018740132454446cb0a2e/docs/Config.md#overriding-default-config-file-location
-    programs.lazygit.settings = catppuccinLib.importYAML (sources.lazygit + "/${cfg.flavor}/${cfg.accent}.yml");
+  config = lib.mkIf enable {
+    home.sessionVariables =
+      let
+        configFiles = [
+          "${sources.lazygit}/${cfg.flavor}/${cfg.accent}.yml"
+        ] ++ lib.optional (config.programs.lazygit.settings != { }) configFile;
```

Comment:
If I understand the issue in #455 correctly, the issue is that home manager doesn't write lazygit's config file if we have an empty attrset.

This will break if someone doesn't use home manager to manage their lazygit config file (and for some reason only manages the theme config with this flake ¯\\\_(ツ)\_/¯ ) but I think it's fine to assume users will manage the config with home manager.